tests/datasets/test_egfxset.py

```diff
@@ -0,0 +1,130 @@
+import os
+from mirdata import annotations
+from mirdata.datasets import egfxset
+from tests.test_utils import run_track_tests
+
+
+def test_track():
+    default_trackid = "TapeEcho_Bridge/2-0"
+    data_home = os.path.normpath("tests/resources/mir_datasets/egfxset")
+    dataset = egfxset.Dataset(data_home, version="test")
+
+    track = dataset.track(default_trackid)
+
+    expected_attributes = {
+        "track_id": "TapeEcho_Bridge/2-0",
+        "audio_path": os.path.normpath(
+            "tests/resources/mir_datasets/egfxset/TapeEcho/Bridge/2-0.wav"
+        ),
+        "stringfret_tuple": [2, 0],
+        "pickup_configuration": "Bridge",
+        "effect": "tape echo",
+        "model": "Line 6 DL4 Delay",
+        "effect_type": "delay",
+        "knob_names": [
+            "effect selector",
+            "delay time",
+            "repeats",
+            "tweak (bass)",
+            "tweez (treble)",
+            "mix",
+        ],
+        "knob_type": ["selector", "rate", "effect decay", "eq", "eq", "effect amount"],
+        "setting": ["tape echo", "120 bpm", 0.6, 0.5, 0.5, 0.5],
+    }
+
+    expected_property_types = {
+        "audio": tuple,
+        "note": annotations.NoteData,
+    }
+
+    assert track._track_paths == {
+        "audio": ["TapeEcho/Bridge/2-0.wav", "bf9041e98fbc3c1145583d1601ab2d7b"]
+    }
+
+    run_track_tests(track, expected_attributes, expected_property_types)
+
+    audio, sr = track.audio
+    assert sr == 48000
+    assert audio.shape == (48000,)
+
+    default_trackid = "Clean_Middle/6-22"
+    track = dataset.track(default_trackid)
+
+    expected_attributes = {
+        "track_id": "Clean_Middle/6-22",
+        "audio_path": os.path.normpath(
+            "tests/resources/mir_datasets/egfxset/Clean/Middle/6-22.wav"
+        ),
+        "stringfret_tuple": [6, 22],
+        "pickup_configuration": "Middle",
+        "effect": "clean",
+        "model": "None",
+        "effect_type": "None",
+        "knob_names": "None",
+        "knob_type": "None",
+        "setting": "None",
+    }
+
+    expected_property_types = {
+        "audio": tuple,
+        "note": annotations.NoteData,
```

Comment:
Small thing: now that note is a `NoteData` object, it would be good to check that its `intervals` and `pitches` values are returning what you expect by simply adding an extra `assert` check for each

Reply:
The asserts were added